test/e2e/config/docker.yaml

```diff
@@ -318,6 +320,19 @@ providers:
       files:
       - sourcePath: "../data/shared/main/metadata.yaml"
 
+- name: helm
+  type: AddonProvider
+  versions:
+  - name: v0.1.0-alpha.4
+    value: https://github.com/kubernetes-sigs/cluster-api-addon-provider-helm/releases/download/v0.1.0-alpha.4/add-on-components.yaml
+    type: url
+    contract: v1beta1
+    files:
+    - sourcePath: "../data/shared/v1beta1/metadata.yaml"
```

Comment:
Sorry for not catching it earlier - you don't need a source path as we're retrieving from a URL here.

Reply:
Mentioned this in slack too -- do we still need to pass the location to a metadata.yaml file then? The URL referenced in the add-on provider is just the add-on-components.yaml file.